indexers/nmslib_indexer.py

```diff
@@ -3,13 +3,10 @@
 import pandas as pd
 from tqdm import tqdm
 tqdm.pandas()
-from text_utils.utils import prepare_ans
-from config import ifile_train_path, index_path
+from utils.base_classes import BaseIndexer
+from config import INDEX_DIR, index_path
 
-MAX_TEXT_LEN = 300
-df = pd.read_csv(ifile_train_path)
-
-class UniIndexer:
+class NMSlibIndexer(BaseIndexer):
     def __init__(self, model_name, logger):
         if model_name == 'bert':
```

Comment:
Better to extract model switch conditions to another function from this indexer class